libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKAILTNPublisher.m

```diff
@@ -65,12 +71,13 @@ + (void) publishLogLines:(NSDictionary *) bodyDictionary publishCompleteBlock:(P
     [request setCustomRequestBodyData:postData contentType:@"application/json"];
     [request setHeaderValue:@"gzip" forHeaderName:@"Content-Encoding"];
     [request setHeaderValue:[NSString stringWithFormat:@"%lu", (unsigned long)[postData length]] forHeaderName:@"Content-Length"];
-    [[SFRestAPI sharedInstance] sendRequest:request failureBlock:^(id response, NSError *e, NSURLResponse *rawResponse) {
+
+    [restAPI sendRequest:request failureBlock:^(id  _Nullable response, NSError * _Nullable e, NSURLResponse * _Nullable rawResponse) {
```

Comment:
Why have nullability markers been added here? They should be added in the definition already, I don't think we need them here.

Reply:
Those got added automatically with code completion, I'll remove them no problem.